individual.cpp

```diff
@@ -49,7 +52,7 @@ void individual::mutate(double mut_rate, double mut_step, std::mt19937_64& rng)
 
 std::vector<double> response(const individual& ind)
 {
-    return response(ind.get_net(),ind.get_input_values(), &sigmoid);
+    return response(ind.get_net(),ind.get_input_values());
```

Comment:
Should the activation not be specified in response though? I understand that forcing it to be sigmoid is not ideal, maybe it should be an argument?

Reply:
the thing is that a network should not change its activation function at run-time, so we should not be able to change it on the fly.
I left a response function that actually allows that though, because it was used in tests, but ideally we would like to get rid of it.

Reply:
Okay. Should we make an issue to take care of that?